src/scripts/movies/FilterRange.ts

```diff
@@ -1,5 +1,23 @@
-export default class FilterRange {
-    constructor(blockId) {
+type TFilterRangeConstructor = {
+    parentRange: Element,
+    infoMin: HTMLInputElement,
+    infoMax: HTMLInputElement,
+    rangeMin: HTMLInputElement,
+    rangeMax: HTMLInputElement
+};
+
+export default class FilterRange implements TFilterRangeConstructor {
+    parentRange: Element;
+
+    infoMin: HTMLInputElement;
+
+    infoMax: HTMLInputElement;
+
+    rangeMin: HTMLInputElement;
+
+    rangeMax: HTMLInputElement;
```

Comment:
if inside the class, then a separate type is not needed

Reply:
Ok